core/services/keystore/vrf.go

```diff
@@ -157,7 +164,7 @@ func (ks *vrf) GetV1KeysAsV2(password string) (keys []vrfkey.KeyV2, _ error) {
 func (ks *vrf) getByID(id string) (vrfkey.KeyV2, error) {
 	key, found := ks.keyRing.VRF[id]
 	if !found {
-		return vrfkey.KeyV2{}, fmt.Errorf("unable to find VRF key with id %s", id)
+		return vrfkey.KeyV2{}, errors.Wrapf(ErrMissingVRFKey, "unable to find VRF key with id %s", id)
```

Comment:
Hah, agreed! I was mirroring what was there for P2P keys for consistency. Ideally I'd want to replace all of these errors with a single error type like `KeyNotFoundError` that has the error message e.g

```golang
return vrfkey.KeyV2{}, KeyNotFoundError{message: fmt.Sprintf("unable to find VRF key with id %s", id)}
```

To answer your question of _why_ add this, it's basically to be able to figure out in the GQL handler if the key was not found more reliably, i.e using `errors.Cause`. We could do regex matching on the error message itself instead, but that feels a bit more flimsy since those are usually fluid and can change anytime.

Reply:
I've addressed this here: https://github.com/smartcontractkit/chainlink/pull/5376/commits/a7d3d3cebcdead8c628532bed37c5b25d10d2caa. Will port it over to this branch.